app/assets/stylesheets/application.scss

```diff
@@ -20,10 +20,11 @@
 @import "nav";
 @import "dashboard";
 @import "footer";
+@import "hamburgermenu";
```

Comment:
you dont need this here - the `_hamburgermenu.html.slim` is a view partial not a stylesheet